alf/examples/ppo2_cart_pole.gin

```diff
@@ -0,0 +1,28 @@
+include 'ppo2.gin'
+
+# environment config
+create_environment.env_name="CartPole-v0"
+create_environment.num_parallel_environments=8
+
+# algorithm config
+PPOLoss2.entropy_regularization=1e-4
+PPOLoss2.gamma=0.98
+PPOLoss2.td_error_loss_fn=@element_wise_huber_loss
+PPOLoss2.normalize_advantages=False
+create_ac_algorithm.actor_fc_layers=(100,)
+create_ac_algorithm.value_fc_layers=(100,)
+create_ac_algorithm.learning_rate=0.001
```

Comment:
Add rewarding shaping makes the training much more stable and faster:
`ActorCriticAlgorithm.reward_shaping_fn=@reward_scaling
  common.reward_scaling.scale=0.01`